plugins/SkyCultureMaker/src/types/SkyPoint.hpp

```diff
@@ -37,13 +37,13 @@ struct SkyPoint
 	Vec3d coordinate = Vec3d(0, 0, 0);
 
 	//! The optional star at that coordinate.
-	std::optional<QString> star = std::nullopt;
+	QString star = QString("");
```

Comment:
This can just be left default-initialized as `QString star;`, no need to pass an empty string to it.